hack/3.11deploy.sh

```diff
@@ -13,7 +13,19 @@
 # See the License for the specific language governing permissions and
 # limitations under the License.
 
-for FILE in deploy/crds/app.kiegroup.org_kogitoapps_crd.yaml deploy/crds/app.kiegroup.org_kogitodataindices_crd.yaml deploy/role.yaml deploy/service_account.yaml deploy/role_binding.yaml deploy/operator.yaml
+# Define CRD files
+crd_folder="deploy/crds"
+crd_files=("app.kiegroup.org_kogitoapps_crd.yaml" "app.kiegroup.org_kogitodataindices_crd.yaml" "app.kiegroup.org_kogitoinfras_crd.yaml")
```

Comment:
Won’t be better to scan the directory for CRD files instead of having them fixed here?

Reply:
We could but I found it more complex to code that's it XD

Reply:
Just add a ==~ to compare regexp looking for “crd” pattern. Hahahah

Reply:
updated :)